Circles/Views/Login/AppHelpView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppHelpView.swift
+//  Circles
+//
+//  Created by Dmytro Ryshchuk on 8/15/24.
+//
+
+import SwiftUI
+import MarkdownUI
+
+struct AppHelpView: View {
+    
+    let helpTextMarkdown = """
+        # Our mission
+                
+        We wanted the social features of Facebook with the security and privacy of Signal, so we built Circles.
+        
+        Circles helps you stay connected with family, friends, and your community, all with the safety of end-to-end encryption.
+        
+        Share your moments, thoughts, and memories safely with those closest to you, without worrying about who else can see.
+        
+        Post to your timeline to share with all of your people in one place.
+        
+        Connect with neighbors, coworkers, and friends from every sphere of life.
+        
+        Circles doesn't track you and doesn't show you ads. We work for you, not for advertisers.
+        Your data belongs to you. You can never be locked in to a single provider.
+        
+        All posts in Circles are encrypted using the same cryptography from state-of-the-art secure messaging tools like Signal and Matrix.
+        """
+    
+    var body: some View {
+        ScrollView {
+            HStack {
```

Comment:
These 4 images were used for the circles tab "help" popup in the past, to illustrate that you can use a circle to share with lots of different people who may not know each other.  (As opposed to a group, where we used a big group photo in the help popup, to illustrate that it's for people who are all connected to each other.)

I don't think it really communicates much about the app here.  We should probably find another "happy family" type picture to use instead.

Reply:
I decided to stay with those images because I know that it's free and we can reuse. Let's discuss what we can use and where I can find it.